internal/postgres/paymentcodes.go

```diff
@@ -0,0 +1,63 @@
+package postgres
+
+import (
+	"context"
+	"database/sql"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/ngavinsir/golangtraining"
+	"github.com/pkg/errors"
+)
+
+type paymentCodesRepository struct {
+	DB *sql.DB
+}
+
+func NewPaymentCodesRepository(db *sql.DB) golangtraining.IPaymentCodesRepository {
```

Comment:
let's practice "accept interface, return struct"
and declare interface at callsite